src/finch/symbolic/rewriters.py

```diff
@@ -169,9 +181,7 @@ def __call__(self, x: Term) -> Term | None:
         if y is not None:
             if y.is_expr():
                 y_args = y.children()
-                return y.make_term(
-                    y.head(), *[default_rewrite(self(arg), arg) for arg in y_args]
-                )
+                return y.make_term(*(default_rewrite(self(arg), arg) for arg in y_args))
```

Comment:
Let's keep `head` here and in the rest of the file.

Reply:
Done in https://github.com/finch-tensor/finch-tensor-lite/pull/17/commits/56349b0d36760ac0f6f916d9b851a2d6744caddd